workshop-registration.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+    <meta charset="UTF-8">
+    <title>WebDev Workshop Registration</title>
+    </head>
+    <body>
+        <h1 id="main-title">WebDev Workshop Registration</h1>
+        <p id="desc1">
+        Join our immersive workshop to sharpen your modern web development skills.<br>
+        Open to all, from beginners to experts—select sessions that match your passion and goals!
+        </p>
+        <form action="https://formspree.io/f/your-form-id" method="POST">
+            <fieldset>
+                <legend>Personal Information</legend>
+                <label for="full-name">Full Name</label><br>
+                <input type="text" id="full-name" name="fullName" required aria-label="Full Name"><br><br>
+                <label for="email">Email</label><br>
+                <input type="email" id="email" name="email" required aria-label="Email"><br><br>
```

Comment:
Why do we need the `aria-label` on the inputs? We already the label and this field associated for for-id pair